tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["configs/**/*", "test/**/*"],
+  "include": ["configs/**/*", "test/**/*", "workflows/**/*"],
```

Comment:
nit: can it be named `scripts/` instead? open to suggestions

Reply:
Done 06f86bc - don't have strong feelings about this directory name